src/components/forms/schemas/PectraPGR.ts

```diff
@@ -0,0 +1,76 @@
+import * as z from 'zod';
+
+import { stringFieldSchema } from './utils'
+import { containURL } from '../../../utils'
+import { MAX_PROPOSAL_FILE_SIZE } from '../../../constants'
+
+const MAX_TEXT_LENGTH = 255
+const MAX_TEXT_AREA_LENGTH = 2000
+const MIN_TEXT_AREA_LENGTH = 500
+const TEXT_AREA_LONG_LENGTH = 32768
+
+const ACCEPTED_FILE_TYPES = ['application/pdf']
+
+export const PectraPGRSchema = z.object({
+  firstName: stringFieldSchema('First name', { min: 1, max: 40 }).refine(
+    value => !containURL(value),
+    'First name cannot contain a URL'
+  ),
+  lastName: stringFieldSchema('Last name', { min: 1, max: 80 }).refine(
+    value => !containURL(value),
+    'Last name cannot contain a URL'
+  ),
+  email: z.string().email({ message: 'Invalid email address' }),
+  individualOrTeam: stringFieldSchema(
+    'Please select whether you are applying as an individual or a team.',
+    { min: 1, max: MAX_TEXT_LENGTH }
+  ),
+  company: stringFieldSchema('Company name', { max: MAX_TEXT_LENGTH })
+    .refine(
+      (data) => data !== undefined && data.trim() !== '',
+      { message: 'Organization name is required' }
```

Comment:
If you set Individual, this field should not be checked. Currently, it is still throwing an error.

![image](https://github.com/user-attachments/assets/83b7a7cb-030e-4bd5-88f5-9a6b83071511)

Since this field is conditional, we need to set this refine at the schema level not at the field level (1 level up).

```suggestion
z.object({
    name: ...,
    email: ...,
}).refine(
    data => ...
);
```

you can see the AGR schema as an example https://github.com/ethereum/esp-website/blob/pectra-pgr/src/components/forms/schemas/AcademicGrants.ts#L69